core/src/main/scala/kafka/server/LogDirFailureChannel.scala

```diff
@@ -29,7 +32,7 @@ import java.util.concurrent.{ArrayBlockingQueue, ConcurrentHashMap}
  * can take the name of the new offline log directory out of the LogDirFailureChannel and handles the log failure properly.
  *
  */
-class LogDirFailureChannel(logDirNum: Int) {
+class LogDirFailureChannel(logDirNum: Int) extends Logging {
 
   private val offlineLogDirs = new ConcurrentHashMap[String, String]
   private val logDirFailureEvent = new ArrayBlockingQueue[String](logDirNum)
```

Comment:
It was named as event because we may expand it later to include more information than the log directory name. I have updated the code to rename methods as suggested.